ci/scripts/build-other.sh

```diff
@@ -7,7 +7,7 @@ source ci/scripts/common.env.sh
 
 echo "--- Build Java connector node"
 # clone a released version(tag)
-git clone --branch v0.0.1 --depth 1 https://"$GITHUB_TOKEN"@github.com/risingwavelabs/risingwave-connector-node.git
+git clone --branch v0.0.2 --depth 1 https://"$GITHUB_TOKEN"@github.com/risingwavelabs/risingwave-connector-node.git
```

Comment:
Can we set a commit id and checkout the commit id instead?

Reply:
Currently, the connector node is closed source. And I release the jar to a public [repo](https://github.com/risingwavelabs/risingwave-connector-release) for users. I think a readable version number is intuitive to maintain the released package.